examples/iotc_roughtime_client/src/iotc_roughtime_client.c

```diff
@@ -0,0 +1,30 @@
+#include <iotc_roughtime_client.h>
+
+#include "string.h"
+#include <inttypes.h>
+#include <stdint.h>
+#include <stdio.h>
+
+#include "../../common/src/example_utils.h"
+
+/* Application variables. */
+iotc_crypto_key_data_t iotc_connect_private_key_data;
+
+int main() {
+  const char* name = "Cloudflare-Roughtime";
+  const char* public_key = "gD63hSj3ScS+wuOeGrubXlq35N1c5Lby/S+T7MNTjxo=";
+  const char* server_address = "roughtime.cloudflare.com:2002";
+  iotc_roughtime_state_t state;
+  iotc_roughtime_timedata_t* time_data =
```

Comment:
I'd use this as a simple stack variable:
`iotc_roughtime_timedata_t* time_data;'

Reply:
Without any initialization, errors of using uninitialized variables. So I just set as NULL...

Reply:
I've changed the variable as struct itself not a pointer.
`iotc_roughtime_timedata_t time_data;`